util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
+    try:
+        if value is None:
+            return default
+        if value.lower() == 'true':
+            return True
+        elif value.lower() == 'false':
+            return False
+        else:
+            return default
+    except Exception as e:
+        return default
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rkj6124/test/pull/2#issuecomment-2399609989>#b5d3aa</a><span> - 10/08/2024, 05:15 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The function catches all exceptions indiscriminately, which may mask unexpected errors and make debugging difficult.
</div>
<div id="fix">
<b>Fix:</b> Catch only specific exceptions that are expected to occur during normal operation of the function. For this function, catching ValueError would be appropriate.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,2 +11,2 @@
 -    except Exception as e:
 -        return default
 +    except ValueError:
 +        return default
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged